bubblewrap.c

```diff
@@ -618,7 +624,7 @@ setup_newroot (bool unshare_pid,
         }
 
       if (op->dest &&
-          op->type != SETUP_SET_HOSTNAME)
+          !op->flags & NO_CREATE_DEST)
```

Comment:
This looks wrong, you need parantesis becaus ! has precedence over &.
I recommend `(op->flags & NO_CREATE_DEST) != 0`